S4TF_Tutorial_1.ipynb

```diff
@@ -541,11 +541,11 @@
         "* Many of the exercises in the course executes more quickly by using GPU runtime: Runtime | Change runtime type | Hardware accelerator | GPU\n",
         "\n",
         "**Some final words on Colab**\n",
-        "*   You execute each cell in order, you can edit & re-execute cells if you want\n",
+        "*   You execute each cell in order, you can edit & re-execute cells if you want.\n",
         "*   Sometimes, this could have unintended consequences. For example, if you add a dimension to an array and execute the cell multiple times, then the cells after may not work. If you encounter problem reset your environment:\n",
```

Comment:
```suggestion
        "*   Sometimes, this could have unintended consequences. For example, if you add a dimension to an array and execute the cell multiple times, the cells that follow may not work. If you encounter problems, reset your environment:\n",
```